contrib/clboss-earnings-history

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+
+import subprocess
+import argparse
+import json
+from datetime import datetime
+from tabulate import tabulate
+
+def run_lightning_cli_command(network_option, command, *args):
+    try:
+        result = subprocess.run(['lightning-cli', network_option, command, *args], capture_output=True, text=True, check=True)
+        return json.loads(result.stdout)
+    except subprocess.CalledProcessError as e:
+        print(f"Command '{command}' failed with error: {e}")
+    except json.JSONDecodeError as e:
+        print(f"Failed to parse JSON from command '{command}': {e}")
+    return None
+
+def format_bucket_time(bucket_time):
+    if bucket_time == 0:
+        return "Legacy"
+    else:
+        return datetime.utcfromtimestamp(bucket_time).strftime('%Y-%m-%d')
+
+def main():
+    parser = argparse.ArgumentParser(description="Run lightning-cli with specified network")
+    parser.add_argument('--mainnet', action='store_true', help='Run on mainnet')
+    parser.add_argument('--testnet', action='store_true', help='Run on testnet')
+    parser.add_argument('nodeid', nargs='?', help='The node ID to pass to clboss-earnings-history (optional)')
+
+    args = parser.parse_args()
+
+    if args.testnet:
+        network_option = '--testnet'
```

Comment:
do we need to support --signet as well?

Reply:
Oh good point ... fixing ...

Reply:
I made it the same as `lightning-cli`, so:
- `clboss-earnings-history`: defaults to mainnet
- `clboss-earnings-history --signet`: use signet
- `clboss-earnings-history --network=signet`

TIL: lightning-cli's network option seems a little inconsistent, the following work:
- `lightning-cli --mainnet`
- `lightning-cli --network=bitcoin`
... but the following does not:
- `lightning-cli --network=mainnet`

Anyway, the clboss scripts attempt to be exactly the same as lightning-cli ...